bindings/go/descriptor/normalisation/engine/jcs/normalise.go

```diff
@@ -0,0 +1,431 @@
+package jcs
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"maps"
+	"slices"
+	"strconv"
+
+	"github.com/cyberphone/json-canonicalization/go/src/webpki.org/jsoncanonicalizer"
+)
+
+// Normalise is a helper function that prepares and marshals a normalized value.
+// It takes an input value and exclusion rules, and returns the canonicalized JSON representation.
+//
+// Parameters:
+//   - v: The input value to normalize
+//   - ex: Exclusion rules to apply during normalization
```

Comment:
From the `doc.go` it is described that `Normalise` can also take a `mapper` which contains a transformation. This is not the same as the exclusion rules, is it?

Reply:
actually MapValue is also an exclusion rule, which is... unfortunate now that I think about it. Maybe they should be called transformation rules instead?

Reply:
I changed the names to transformationRules instead of exclude rules

Reply:
sounds reasonable